index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
```

Comment:
This is the part of Technical Onboarding.
[README.md](https://github.com/AfraSadat/introduction-to-github/files/10146504/README.md)